src/dataloader.ts

```diff
@@ -19,6 +19,7 @@ const languages = {
   ".R": ["Rscript"],
   ".rs": ["rust-script"],
   ".go": ["go", "run"],
+  ".cmd": ["cmd", "/c"],
```

Comment:
This would be nice as a standalone PR.

Reply:
See:  https://github.com/observablehq/framework/pull/969